### ISSUE TITLE
Incompatible with Rails 7.1.0

### ISSUE THREAD

**Issue by TBenedyk** [veeqo/activejob-uniqueness#64]
I'm getting this error when trying to upgrade rails from `7.0.8` to `7.1.0`.
```
➜  my_app git:(main) ✗ bundle update rails
Fetching gem metadata from https://rubygems.org/........
Resolving dependencies...
Could not find compatible versions

Because activejob-uniqueness >= 0.1.3, < 0.2.3 depends on activejob >= 4.2, < 7
  and activejob-uniqueness >= 0.2.3 depends on activejob >= 4.2, < 7.1,
  activejob-uniqueness >= 0.1.3 requires activejob >= 4.2, < 7.1.
Because rails >= 7.1.1 depends on activejob = 7.1.1
  and rails >= 7.1.0, < 7.1.1 depends on activejob = 7.1.0,
  rails >= 7.1.0 requires activejob = 7.1.0 OR = 7.1.1.
Thus, activejob-uniqueness >= 0.1.3 is incompatible with rails >= 7.1.0.
So, because Gemfile depends on rails ~> 7.1.0
  and Gemfile depends on activejob-uniqueness ~> 0.2,
  version solving has failed.
➜  shopify_app git:(main) ✗
```
It seems like this gem restricts itself to activejob versions `< 7.1` here https://github.com/veeqo/activejob-uniqueness/blob/96e207acdb8ca7c16aaf68676d36b358672ac480/activejob-uniqueness.gemspec#L31

Is there a reason for this or would a simple fix to remove this restriction be OK? I haven't ran the specs without it to be sure.

**Comment by laurafeier** [comment]
I opened #66 #65 to get this moving.

**Comment by sharshenov** [comment]
Resolved by #66
